tests/FormatterMock.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
There is no need to create abstract test case class. Make separated class-factory for this purpose.

Reply:
Why it was resolved without any comments?

Reply:
resolved